src/mesh.cpp

```diff
@@ -453,7 +459,8 @@ scene::IMesh* convertNodeboxNodeToMesh(ContentFeatures *f)
 		f32 tx2 = (box.MaxEdge.X/BS)+0.5;
 		f32 ty2 = (box.MaxEdge.Y/BS)+0.5;
 		f32 tz2 = (box.MaxEdge.Z/BS)+0.5;
-		f32 txc[24] = {
+
+		f32 txc_default[24] = {
```

Comment:
All this talk about "txc", care to explain what a txc is, or better yet, name your parameters better to begin with?